machine.go

```diff
@@ -552,7 +556,9 @@ func (m *Machine) call(ctx context.Context, serviceMethod string, arg, reply int
 			}
 		}()
 	}
-	return m.client.Call(ctx, m.Addr, serviceMethod, arg, reply)
+	err = m.client.Call(ctx, m.Addr, serviceMethod, arg, reply)
+	log.Printf(">>>>>> called: %v: %v", serviceMethod, err)
+	return err
```

Comment:
debug printf.

btw, if the grail logger is configured with level=debug, the bigmachine RPC stack will log all calls.